doc/api/cli.md

```diff
@@ -594,6 +595,18 @@ added: v0.11.15
 
 Specify ICU data load path. (Overrides `NODE_ICU_DATA`.)
 
+### `--import=module`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Preload the specified module at startup.
+
+Follows [ECMAScript module][] resolution rules.
+Use [`--require`][] to load a [CommonJS module][].
+modules preloaded with `--require` will run before modules preloaded with `--import`
```

Comment:
We should have tests for this (that can happen in a follow-up PR though).
```suggestion

Modules preloaded with `--require` will run before modules preloaded with `--import`.
```

Reply:
tests failed do to a missing await, so added a test, please re approve